lib/metadata/BucketInfo.js

```diff
@@ -2,7 +2,7 @@ import assert from 'assert';
 import { WebsiteConfiguration } from './WebsiteConfiguration';
 
 // WHEN UPDATING THIS NUMBER, UPDATE MODELVERSION.MD CHANGELOG
-const modelVersion = 4;
+const modelVersion = 5;
```

Comment:
It looks like this got messed up in the forward port since the md document already says version 5.  Will you please do a separate cleanup pr that just updates this number?